keras/applications/applications_test.py

```diff
@@ -191,6 +191,14 @@ def test_application_pooling(self, app, last_dim):
         )
         self.assertShapeEqual(output_shape, (None, last_dim))
 
+    @parameterized.parameters(MODEL_LIST)
+    def test_application_classifier_activation(self, app):
```

Comment:
The unit tests are failing, since MODEL_LIST produce two items.

```
ERROR: test_application_classifier_activation53 (<function RegNetY008 at 0x7f760cb75310>, 768) (__main__.ApplicationsTest)
ApplicationsTest.test_application_classifier_activation53 (<function RegNetY008 at 0x7f760cb75310>, 768)
test_application_classifier_activation(<function RegNetY008 at 0x7f760cb75310>, 768)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmpfs/venv/lib/python3.9/site-packages/absl/testing/parameterized.py", line 320, in bound_param_test
    return test_method(self, *testcase_params)
TypeError: test_application_classifier_activation() takes 2 positional arguments but 3 were given

----------------------------------------------------------------------
Ran 11 tests in 62.691s

FAILED (errors=2)
```

Reply:
This has been rectified now by adding an extra/unused param `_`